Visualizer/src/components/home/owner/OwnerForm.vue

```diff
@@ -29,30 +29,41 @@ const props = defineProps({
 const simulationConfig = useSimulationConfigStore();
 
 const owner = computed(() => simulationConfig.owners[props.ownerIndex]);
-
 loadCompatibleValueFunctions();
+
+const biddingStrategy = ref(owner.value.biddingStrategy.classname);
 /**
  * Whenever the selected ownerType changes, make sure the requirements for minimum
  * and maximum number of locations are met
  */
-const updateLocationsForOwner = () => {
+
+const compatibleValueFunctions = ref([]);
+function loadCompatibleValueFunctions() {
+  getSupportedValueFunctions(simulationConfig.allocator, owner.value.biddingStrategy.classname).then((res) => {
+    compatibleValueFunctions.value = res.map((a) => ({ label: a["label"], value: a["classname"] }));
```

Comment:
You could use a.label and a.classname syntax here since it is javascript and not python